src/Gui/ConversionUtils.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Pytocs.Translate;
+using Pytocs.TypeInference;
+
+namespace Pytocs.Gui
+{
+    public static class ConversionUtils
+    {
+        public static async Task ConvertFolder(string sourcePath, string targetPath, ILogger logger)
```

Comment:
Prefer ending async methods with the fragment `Async` (e.g. here, change to `ConvertFolderAsync`).

Also, please add a doc comment explaining what purpose the method serves, and why It can't be in the `Translator` class.